verticapy/plotting/_plotly/base.py

```diff
@@ -16,6 +16,40 @@
 """
 from verticapy.plotting.base import PlottingBase
 
+import numpy as np
+
 
 class PlotlyBase(PlottingBase):
-    ...
+    @staticmethod
```

Comment:
will you use this method in many classes or just the nested pie?

Reply:
for now, it is only nested pie.

Reply:
ok maybe you should move this function to your plotly base class as it seems like a general method.